DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/Shared/DelegateInfoViewModel.cs

```diff
@@ -1,49 +1,52 @@
-﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.Shared
 {
     using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Enums;
     using DigitalLearningSolutions.Data.Models.User;
     using DigitalLearningSolutions.Web.ViewModels.Common;
 
     public class DelegateInfoViewModel
     {
-        public DelegateInfoViewModel(DelegateUserCard delegateUser, List<CustomFieldViewModel> customFields)
+        public DelegateInfoViewModel(DelegateUserCard delegateUser, IEnumerable<CustomFieldViewModel> customFields)
         {
             Id = delegateUser.Id;
             Name = delegateUser.SearchableName;
             CandidateNumber = delegateUser.CandidateNumber;
-
-            IsSelfReg = delegateUser.SelfReg;
-            IsExternalReg = delegateUser.ExternalReg;
+            
             IsActive = delegateUser.Active;
-            IsAdmin = delegateUser.AdminId.HasValue;
-            IsPasswordSet = delegateUser.Password != null;
+            IsAdmin = delegateUser.IsAdmin;
+            IsPasswordSet = delegateUser.IsPasswordSet;
+            RegistrationType = delegateUser.RegistrationType;
 
             Email = delegateUser.EmailAddress;
+            JobGroupId = delegateUser.JobGroupId;
             JobGroup = delegateUser.JobGroupName;
             if (delegateUser.DateRegistered.HasValue)
             {
                 RegistrationDate = delegateUser.DateRegistered.Value.ToString("dd/MM/yyyy");
             }
+
             AliasId = delegateUser.AliasId;
 
-            CustomFields = customFields;
+            CustomFields = customFields.ToList();

```

Comment:
Given that the type is changed to IEnumerable do we need the ".ToList()" conversion?